policyengine_us/variables/gov/states/in/tax/income/exemptions/in_base_exemptions.py

```diff
@@ -12,4 +12,14 @@ class in_base_exemptions(Variable):
 
     def formula(tax_unit, period, parameters):
         p = parameters(period).gov.states["in"].tax.income.exemptions
-        return tax_unit("tax_unit_size", period) * p.base.amount
+        # Personal exemptions
+        non_dependent_exemptions = p.base.amount
+        # Dependent exemptions
+        dependents = tax_unit("tax_unit_dependents", period)
+        unit_size = tax_unit("tax_unit_size", period)
+        unit_without_dependents = max_(unit_size - dependents, 0)
+        total_base = unit_without_dependents * non_dependent_exemptions
```

Comment:
Please rename `base/amount` to `base/filer`. Also please avoid adding new Python variables that only rename parameters, as the absence of `p.` makes it less obvious that it's a parameter.
```suggestion
        total_base = unit_without_dependents * p.base.filer
```